meta-mender-demo/recipes-mender/mender/mender_%.bbappend

```diff
@@ -4,7 +4,7 @@ SRC_URI_append = " file://server.crt"
 
 MENDER_UPDATE_POLL_INTERVAL_SECONDS = "5"
 MENDER_INVENTORY_POLL_INTERVAL_SECONDS = "5"
-MENDER_RETRY_POLL_INTERVAL_SECONDS = "1"
+MENDER_RETRY_POLL_INTERVAL_SECONDS = "10"
```

Comment:
Nice and simple, but I think we've been considering increasing this to 30 seconds.